### ISSUE TITLE
{multiple-input} pattern change not enough

### ISSUE THREAD

**Issue by lorinczelod** [unclead/yii2-multiple-input#46]
If I use the kartik\typeahead\TypeAhead widget with the MultipleInput widget,
the TypeAhead generates, for example, the next JS code:
```
var persons_title_{multiple_index}_data_1 = new Bloodhound({"datumTokenizer":Bloodhound.tokenizers.obj.whitespace('value'),"queryTokenizer":Bloodhound.tokenizers.whitespace,"remote":{"url":"/basic/web/persons/country-list?q=QUERY","wildcard":"QUERY"}});
kvInitTA('persons_title-{multiple-index}', typeahead_7864e59a, [{"display":"value","name":"persons_title_{multiple_index}_data_1","source":persons_title_{multiple_index}_data_1.ttAdapter()}]);
```
As you see here the id (ex. persons_title-{multiple-index}) is converted to variable name (persons_title_{multiple_index}_data_1), but is not changed the multiple_index to the new index in 'addInput' function.
What do you think about this?

**Comment by lorinczelod** [comment]
I use this change in 'jquery.multipleInput.js' to fix this problem
```
 for (i in settings.jsTemplates) {
                jsTemplate = settings.jsTemplates[i].replaceAll('{multiple-index}', data.currentIndex);
                jsTemplate = jsTemplate.replaceAll('{multiple_index}', data.currentIndex); // new code
                window.eval(jsTemplate);
            }
```

**Comment by unclead** [comment]
I think its better to use `multiple_input` key everywhere. Name `multiple-input` uses only for internal aims
Want to make a pr?

**Comment by lorinczelod** [comment]
Yes, I want, but I don't know how can I make PR. Could you show me? Do you have a chat?

**Comment by unclead** [comment]
You can use yii2 git workflow: https://github.com/yiisoft/yii2/blob/master/docs/internals/git-workflow.md

**Comment by lorinczelod** [comment]
Thank you your guidance!

2015-10-13 15:45 GMT+03:00 Eugene Tupikov <notifications@github.com>:

**Comment by unclead** [comment]
Closed via 12b8de40acf4e31c9978819cfba9442e767af21b
